sympy/geometry/util.py

```diff
@@ -657,14 +654,21 @@ def intersection(*entities):
     Reals should be converted to Rationals, e.g. Rational(str(real_num))
     or else failures due to floating point issues may result.
 
+    Case 1: When the keyword argument 'pairwise' is False (default value):
+    In this case, the functon returns a list of the points
+    through which all the entities pass.
+
+    Case 2: When the keyword argument 'pairwise' is True:
+    In this case, the functions returns a list of all points
+    where intersections take place.(Please Note that these points
```

Comment:
...where intersections take place.(Please Note that these points
may not necessarily lie on all the entities)

`-->`

...where intersections take place.  A given intersection may not necessarily
lie on all the entities.

You should probably avoid use of the word "point", too, since the intersection of two segments may be a segment.